app/view_models/account_show.rb

```diff
@@ -148,7 +148,7 @@ def recent_event_partial
   def header_personalization
     return decrypted_pii.first_name if decrypted_pii.present?
 
-    decorated_user.email
+    EmailContext.new(decorated_user.user).last_sign_in_email_address.email
```

Comment:
Believe it or not this doesn't have the proper covered index on it.  There needs to be a user_id, last_sign_in_at where confirmed_at is not null but I guess postgres is playing it cool with the plan and still using user_id, last_sign_in_at.